src/Advection/upwind_biased_fifth_order.jl

```diff
@@ -3,11 +3,11 @@
 #####
 
 """
-    struct UpwindBiasedFifthOrder <: AbstractUpwindBiasedAdvectionScheme{2}
+    struct UpwindBiasedFifthOrder <: AbstractUpwindBiasedAdvectionScheme{3}
 
 Upwind-biased fifth-order advection scheme.
 """
-struct UpwindBiasedFifthOrder <: AbstractUpwindBiasedAdvectionScheme{2} end
+struct UpwindBiasedFifthOrder <: AbstractUpwindBiasedAdvectionScheme{3} end
```

Comment:
Why are you changing the definition of the boundary buffer in this PR?

Reply:
The number "2" is the number of buffer cells over which we limit to second order advection. Therefore `CenteredSecondOrder` has a buffer of "0" by definition.

We can change the definition of this number if we want, but some rationale might be helpful...